docs/application-connector/docs/001-overview-application-connector.md

```diff
@@ -3,16 +3,30 @@ title: Overview
 type: Overview
 ---
 
-The Application Connector is a proprietary Kyma implementation that allows you to connect with external solutions. The Application Connector consists of four
-components that ensure the security of the connection and the access to all of the external solution's Events and APIs. The implementation handles routing of the calls and Events coming from an external solution to Kyma, and the API calls sent from Kyma to the connected external solution.
+The Application Connector is a part of the Kyma project which simplifies integration of various systems with Kyma.
 
-These are the components of the Application Connector:
+No matter if you want to integrate an on-premise or a cloud system, the Application Connector ensures that integration works in the same way and developers are not distracted by the configuration or networking issues.
 
-- The **Connector Service** generates the required certificates and ensures a secure and trusted connection between Kyma and an external solution. This is a global service that works in the context of
-a given Remote Environment.
-- The **Metadata Service** allows you to register all of the external solution's APIs and Event catalogs which Kyma consumes. You can register the APIs along with additional documentation and Swagger files.
-This is a global service that works in the context of a given Remote Environment.
-- The **Gateway Service** proxies the API calls sent from Kyma to the connected external solution and handles OAuth2 tokens. A new instance of this service is deployed for every Remote Environment.
-- The **Event Service** delivers the Events sent from a connected external solution to Kyma. A new instance of this service is deployed for every Remote Environment.
+The Application Connector ensures that a connected system communicates with Kyma securely using the client certificate which can be acquired from the Connector service. A client certificate ensures that each connected system is separated.
```

Comment:
instead of writing `The Application Connector` 5 times, better write it once with some good intro text and then mention all those features/advantages in a bullet point list